File: Mobile/D2/LambdaPP_AndroidApp_D2/app/src/main/java/com/example/lambdapp_androidapp_d2/models/Equipment.java
package com.example.lambdapp_androidapp_d2.models;

import java.util.List;

public class Equipment {
    public int eqp_ID;
    public String eqp_Name;
    public String eqp_Description;
    public int eqp_Quantity_Total;
    public boolean eqp_Enabled;
    public List<Task_Equipment_Bridge> task_Equipment_Bridges;
    public List<Task> tasks;
}

File: Dev/Frontend/LambdaPP_WebApp_Alpha_3.0/WildTask/src/app/logic/models/Equipment.ts
export default class Equipment
{
    eqp_ID!: number
    eqp_Name!: string
    eqp_Description!: string
    eqp_Quantity_Total!: number
    eqp_Image!: string
    eqp_Enabled!: boolean;

    constructor(name: string, desc: string, quantity: number)
    {
        this.eqp_Name = name
        this.eqp_Description = desc
        this.eqp_Quantity_Total = quantity
        this.eqp_Enabled = true;
    }
}